Dockerfile

```diff
@@ -1,17 +1,21 @@
-FROM quay.io/fedora/fedora:38
+FROM golang:1.21 as builder
```

Comment:
I believe this is a dependency on dockerhub which we are trying to avoid (rate limits unless $$ and the like)

Reply:
i wasn't aware of it, very good to know, thanks. We'd like to run kubevirt-disk-uploader pod inside KubeVirt Tekton Tasks (to extract and upload windows images).

Reply:
Better stick with quay / fedora-minimal for now

Reply:
Yes, I'll change base image in a new PR https://github.com/codingben/kubevirt-disk-uploader/pull/7.